utils/m0reportbug

```diff
@@ -709,7 +681,7 @@ m0traces() {
         ## Kernel buffer.
         if [ -x "$m0trace" ] && lsmod | grep -q '^m0ctl\b'; then
             mkdir $outdir
-            $m0trace kernel-buf -Y | _xz >$outdir/kernel-buf.yaml.xz
+            $m0trace kernel-buf -Y 2>>$REPORT | _xz >$outdir/kernel-buf.yaml.xz
```

Comment:
If by any chance m0trace fails because of some other issue, then we would not be able to understand why it is failing, 
Can we collect the error in another variable, output the to terminal if it is failed.

Reply:
I think we can remove this part now. 
I see this error every time now since we have removed debugfs for opensource. 
and there is no point to collects kernel-buf because we won't be getting hence onwards.

Reply:
mount | grep m0t1fs
Get this info only if above command show the m0fs mount.

Reply:
@madhavemuri  Updated the patch as you mention, but I am quite unsure, will we have kernel buffer in case of m0t1fs client. 

Because to test this I tried to collect kernel-buffer (m0trace kernel-buf -Y | less) while m0t1fs is mounted during m0t1fs_dgmode_io.sh test run, but still I observed similar error.
"m0tracedump: Failed to open input file '/sys/kernel/debug/motr/trace/buffer': No such file or director"

Reply:
@yatin-mahajan 
Can you please ignore the following error in script:
```
"m0tracedump: Failed to open input file '/sys/kernel/debug/motr/trace/buffer': No such file or director"
```

The '/sys/kernel/debug/motr/trace/buffer' is not available at this moment, because we have lack of debugfs support due to NON-GPL kernel module license.

Please skip this '/sys/kernel/debug/motr/trace/buffer' related code or ignore the error in code.

Please refer to "https://jts.seagate.com/browse/EOS-10625"

Reply:
After discussion with Madhav, I have removed it completely.